R/03-geojob-obj.R

```diff
@@ -75,12 +74,10 @@ setMethod("geojob", signature("XMLDocument"), function(xml, ...) {
 
 #'@param xml location of xml (URL or local path) 
 #'@rdname geojob-methods
-#'@importFrom RCurl url.exists
-#'@importFrom XML xmlTreeParse 
 #'@aliases geojob,geojob-method
 setMethod("geojob", signature("character"), function(xml, ...) {
   #parse based on xml class
-  if(url.exists(xml)){
+  if(startsWith(xml, "http")){
     xml <- gGET(xml)
   } 
   doc <- xmlTreeParse(xml)
```

Comment:
I suppose I could check for `www` too.  `url.exists` would be more robust, but I didn't realize we used the `curl` package instead of `RCurl`, so that would add a new dependency.